protocols/autonat/src/behaviour.rs

```diff
@@ -589,6 +589,12 @@ impl NetworkBehaviour for Behaviour {
                 self.inner.on_swarm_event(listener_closed)
             }
             confirmed @ FromSwarm::ExternalAddrConfirmed(_) => self.inner.on_swarm_event(confirmed),
+            FromSwarm::NewExternalAddrOfPeer(NewExternalAddrOfPeer {
+                addr: _addr,
+                peer_id: _peer_id,
+            }) => {
+                todo!();
+            }
```

Comment:
I don't think autonat needs to know about new external addresses of a peer :)

Reply:
:heavy_check_mark: